docs/marketing/marketing.md

```diff
@@ -0,0 +1,8 @@
+# Marketing
+
+Marketing serves as Hubspot’s primary email marketing tool, where users can batch email a list of contacts in the system. Currently, Cal-ITP has access to these features, but doesn’t use these features.
```

Comment:
Looks like these same two sentences are in 2 separate tabs. As it is, it's repetitive but I'm sure we'll build this out more later so maybe it's intentional